connectors/agentic-ai/src/main/java/io/camunda/connector/agenticai/a2a/client/common/sdk/A2aSdkClientConfig.java

```diff
@@ -10,4 +10,14 @@
 import javax.annotation.Nullable;
 
 @AgenticAiRecord
-public record A2aSdkClientConfig(int historyLength, @Nullable Boolean supportPolling) {}
+public record A2aSdkClientConfig(
+    int historyLength, boolean blocking, @Nullable PushNotificationConfig pushNotificationConfig) {
+
+  public A2aSdkClientConfig {
+    if (blocking && pushNotificationConfig != null) {
```

Comment:
It looks like these two are not mutually exclusive in the specifications but I think it makes sense to have this restriction.